### PR TITLE
Update to Babel6

### PR DESCRIPTION
**Pull request by paulfalgout**
Previously I couldn't get the rollup preset to work in the build for v6, but this solved it.

### CODE REVIEW

File: test/setup/setup.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = function() {
   Backbone.$ = jQuery;
   Backbone.ChildViewContainer = require('backbone.babysitter');
   Backbone.Radio = require('backbone.radio');
-  var Marionette = require('../../src/backbone.marionette');
+  var Marionette = require('../../src/backbone.marionette').default;
```
Comment (JSteunou) [inline test/setup/setup.js]: You can get rid of this by using https://github.com/59naga/babel-plugin-add-module-exports
I was super annoying by this when moving to babel v6

**Comment by jasonLaster** [comment]
:+1:

**Comment by paulfalgout** [comment]
wait on a merge here.. I need to re-try the browser test..  I think it was failing, but I didn't notice because the previous test build was loaded in the browser

**Comment by paulfalgout** [comment]
Ok.  I think this guy is ready to go.

File: test/setup/setup.js (hunk 1 above)
Comment (paulfalgout) [inline test/setup/setup.js]: I agree it is annoying.. not sure about needing the plugin at the moment.  If it comes up again for some reason, certainly.

**Comment by ahumphreys87** [comment]
:+1:

**Pull request merged**